### ISSUE TITLE
Server save before shutdown, por attack Ddos, crashing, error w7...etc

### ISSUE THREAD

**Issue by lopez65** [otland/forgottenserver#1546]
Hello Mr. Mark Samman
Sorry to address here without knowing anything, but it's like no one I resoponde in OTLand, as I speak to you directly as it is the wizard of forgotten server, and it will look like a stupid question, and maybe even her attention, but here goes: Why has your compilations removed from the save when you close the server ?, and implement it again?
thanks for your time and master greetings

**Comment by Nu77** [comment]
I don't understand very well what you are trying to ask but if is the shutdown on server save you can have a look in "data/globalevents/scripts/serversave.lua"
```lua
local shutdownAtServerSave = false
local cleanMapAtServerSave = false
````
just set it to true.

**Comment by lopez65** [comment]
I feel that you did not understand the question, plus I already put a topic in OTLand, but I think as we are in holiday period, no one has answered me yet, but really the question was for which compiles TFS, mark Samman, but since I you answered your Null77, I added information, I want to know is: Because before when the server is closed for example by a DDoS attack, before closing, to a server save so no one lost the exp, items, etc .. and now when it closes the server, the server does not save.
I hope now you understand NULL77 friend
greetings and thanks for your answer

**Comment by ranisalt** [comment]
Try this:
https://github.com/otland/forgottenserver/pull/1435

**Comment by marksamman** [comment]
How do you close the server?

**Comment by lopez65** [comment]
marksamman
hi master, to see if I can explain so you understand me, you advance to use google translator xDD I had emulator tfs 1.0 to warm 10.41, and when the server is closed for any errors, before to a save of players and house items, but in your version 1.1 and 1.2 you have not compiled, and I wonder why you have not done and if it will add in future versions of your great TFS work.
I searched OTLand if through talkactions anyone could do but nobody gives me an answer, if you, who placticamente're a genius in this of complilar TFS, can add you would be eternally grateful, because as a security measure and for my players do not lose anything, I had to modify the server normally make a save to save every 3 hours.
Greetings and thanks for your master answer.
